account_reward_history.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+type AccountRewardHistory struct {
+	// The acount this reward history is for
+	Account common.Address `json:"account"`
+
+	// The reward in THIX "gweis" for this account at this moment
+	Rewards uint64 `json:"rewards"`
+
+	// The sum of rewards ever received uptil now
+	TotalRewards uint64 `json:"totalRewards"`
```

Comment:
(Total)Reward are 256 bits numbers. THIX uses 18 decimals, 64 bits won't be be large enough. Use `*big.Int` to track THIX amounts.